contrib/resources/translations/fr.lng

```diff
@@ -1609,7 +1609,7 @@ Il n'est possible d'utiliser SUBST que sur les lecteurs locaux.
 :SHELL_STARTUP_CGA
 ║ DOSBox supporte le mode composite CGA.                                     ║
 ║ Utilisez [31mF12[37m pour régler la sortiecomposite sur ON, OFF, ou AUTO (défaut). ║
-║ [31m(%s+)F11[37m change la teinte; [31m%s+%s+F11[37m sélectionne type de modèle CGA.%s   ║
+║ [color=light-red]F10[color=white] selects the CGA settings to change and [color=light-red](%s+)F11[color=white] changes it.           ║
```

Comment:
The french translation is saying something about a Ctrl+Alt+F11 hotkey.  That's what the extra strings are for: "Ctrl" and "Alt" plus an empty string that appears unused even here.

I can't confirm what Ctrl+Alt+F11 is supposed to do.  On Linux, it flips me to an empty virtual console :laughing: 

I'd recommend removing the extra unused string parameter from the caller.

Reply:
OK, removed - looks like you are right.